N/NEO/build_tarballs.jl

```diff
@@ -83,7 +88,30 @@ dependencies = [
     Dependency("oneAPI_Level_Zero_Headers_jll", v"1.5.8"; compat="1.5.8"),
 ]
 
-# GCC 4 has constexpr incompatibilities
-# GCC 7 triggers: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=79929
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               preferred_gcc_version=v"8")
+augment_platform_block = raw"""
+    using Base.BinaryPlatforms
+
+    # Can't use Preferences since we might be running this very early with a non-existing Manifest
+    NEO_jll_uuid = Base.UUID("700fe977-ac61-5f37-bbc8-c6c4b2b6a9fd")
+    const preferences = Base.get_preferences(NEO_jll_uuid)
+    Base.record_compiletime_preference(NEO_jll_uuid, "debug")
```

Comment:
Am I missing this elsewhere?

Reply:
oneAPI.jl will set this preference. I originally wanted this JLL to depend on a preference in oneAPI.jl, but it looks like (oneAPI.jl not being a dependency of this JLL) that doesn't cause the necessary invalidation when the pref changes.